controllers/v1/feedback.go

```diff
@@ -14,8 +16,28 @@ type FeedbackController struct {
 	FeedbackService feedbaclServices.FeedbackService
 }
 
+func (ctrl FeedbackController) registerValidators() {
+	if err := binding.Validator.RegisterValidation("isvalidquestionresponse",
+		feedbackSerializers.IsValidQuestionResponse); err != nil {
+		fmt.Println(err.Error())
+	}
+	if err := binding.Validator.RegisterValidation("isvalidsubmittedat",
+		feedbackSerializers.IsValidSubmittedAt); err != nil {
+		fmt.Println(err.Error())
+	}
+	if err := binding.Validator.RegisterValidation("isvalidsaveandsubmit",
+		feedbackSerializers.IsValidSaveAndSubmit); err != nil {
+		fmt.Println(err.Error())
+	}
+	if err := binding.Validator.RegisterValidation("allquestionpresent",
+		feedbackSerializers.IsAllQuestionPresent(ctrl.FeedbackService.DB)); err != nil {
```

Comment:
for similar validators, create a validator class which could be called at admin level, also this looks apter at the service level and not serializer, let discuss and move